torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -214,12 +214,32 @@ def parallelize_llama(model, world_mesh, parallel_dims, job_config: JobConfig):
             param_dtype=torch.bfloat16, reduce_dtype=torch.float32
         )
         ac_mode = job_config.activation_checkpoint.mode
+        # specifically compile just the RMSNorm layers
+        enable_compile_rmsnorm = job_config.training.compile_rmsnorm
+        if job_config.training.compile and enable_compile_rmsnorm:
```

Comment:
IMO there's no need to print this message. It's still compiled, although with the whole model.